README.md

```diff
@@ -1,655 +1,119 @@
-Shopify App
-===========
-[![Version][gem]][gem_url] [![Build Status](https://github.com/Shopify/shopify_app/workflows/CI/badge.svg)](https://github.com/Shopify/shopify_app/actions?query=workflow%3ACI)
+# Shopify App
+
+[![Version][gem]][gem_url] [![Build Status](https://github.com/Shopify/shopify_app/workflows/CI/badge.svg)](https://github.com/Shopify/shopify_app/actions?query=workflow%3ACI) ![Supported Rails version][supported_rails_version]
 
 [gem]: https://img.shields.io/gem/v/shopify_app.svg
 [gem_url]: https://rubygems.org/gems/shopify_app
+[supported_rails_version]: https://img.shields.io/badge/rails-%3C6.1.0-orange
```

Comment:
I feel it would be easy to forget to update this when releasing updates for newer Rails versions.